### ISSUE TITLE
Updated bots for 1.11

### ISSUE THREAD

**Issue by lonewolfss** [x1234xx/BotAi-WORKING-SOLDER-s-Dayz-SA#4]
here is a update for the bots that was just added to the workshop use the same server side files that you had previously and add the new clientside files and new key all works 100% 

[@BotClientSide.zip](https://github.com/x1234xx/BotAi-WORKING-SOLDER-s-Dayz-SA/files/6096382/%40BotClientSide.zip)

**Comment by Kilgoreb29** [comment]
Is this the newest version, and where is the Server files?

**Comment by lonewolfss** [comment]
yes its the newest version but i dont have the server side files i currently use the old ones and they work just fine same with the mp missions files they are the same i think im not sure how to get them but the new client side are good

**Comment by Kilgoreb29** [comment]
Checkout the newst version I posted. It was on ragezone, so I uploaded it to here. Says its version 2.06
